cves/CVE-2016-4438.yml

```diff
@@ -207,4 +219,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This is an input validation error. It is an input validation error because a user can
+    enter code to excute arbitraury code and failure with code can lead to a denial of service. This seems
+    to be a classic case of improper input validation because of the lack of input sanitization.
```

Comment:
Did they have any input validation or sanitization beforehand or was it simply incomplete? Was this a design mistake or a coding mistake? A little more detail here might be nice :)

Reply:
I updated this on the last commit